test/unit/bin/ember-template-lint-test.js

```diff
@@ -127,6 +127,17 @@ describe('ember-template-lint executable', function() {
   });
 
   describe('errors and warnings formatting', function() {
+    let oldValue;
+
+    beforeEach(function() {
+      oldValue = process.env.GITHUB_ACTIONS;
+      delete process.env.GITHUB_ACTIONS;
+    });
+
+    afterEach(function() {
+      process.env.GITHUB_ACTIONS = oldValue;
```

Comment:
If `process.env.GITHUB_ACTIONS` was originally not defined, this will define it and make it truthy. 😱

![Screen Shot 2019-12-05 at 4 29 08 PM](https://user-images.githubusercontent.com/12637/70275559-5f459280-177c-11ea-8c07-abcfd57f1757.png)


---

I think we need to do something like:

```
// in module scope, eagerly
const isGithubActions = process.env.GITHUB_ACTIONS;

// ...snip...
beforeEach(function() {
  delete process.env.GITHUB_ACTIONS;
});

afterEach(function() {
  if (isGithubActions) {
    process.env.GITHUB_ACTIONS = 'true';
  } else {
    delete process.env.GITHUB_ACTIONS;
  }
});
```

Reply:
done, I've extracted a reusable `setupEnvVar()` test helper function :)